docs/FEATURES.md

```diff
@@ -0,0 +1,28 @@
+# Features
```

Comment:
Listing all features in a file doesn't seem like a good idea to me. The "projects" tab on github would be a much better place to put those in, in my opinion.

Reply:
Thing is, each time we'll make a PR adding a feature we'll need to think about ticking the box in that file.

Reply:
Projects are for future features, not existing ones imho.

Reply:
When looking at amethyst first I had no idea what it actually did. Only after looking at examples I got it. I am for a list of the _main_ features, but no need to keep everything up to date an 100% accurate at all times.